pybv/io.py

```diff
@@ -228,19 +272,51 @@ def _write_vmrk_file(vmrk_fname, eeg_fname, events, meas_date):
                   f'{i_dur},0', file=fout)
 
 
-def _scale_data_to_unit(data, unit):
-    """Scale `data` in Volts to `data` in `unit`."""
-    scale = SUPPORTED_UNITS.get(unit, None)
-    if scale is None:
-        msg = (f'Encountered unsupported unit: {unit}\n'
-               f'Use one of the following: {set(SUPPORTED_UNITS.keys())}')
-        raise ValueError(msg)
-
-    return data * scale
+def _scale_data_to_unit(data, units):
+    """Scale `data` in Volts to `data` in `units`."""
+    # keep track of any unsupported units to warn user
+    # these are voltage units that we convert to microvolts
+    unsupported_units = set()
+
+    # officially unsupported units that contain non-voltage
+    # related signals
+    official_unsupp_units = set()
```

Comment:
Can you rename these to `voltage_units` and `non_voltage_units`?

Reply:
Officially unsupported and unsupported is not really easy to understand (so maybe also remove this terminology from the comments).

Reply:
Done.

Reply:
Lmk wyt?